MergerLogic/DataTypes/Gpkg.cs

```diff
@@ -42,47 +27,34 @@ public class Gpkg : Data<IGpkgUtils>
             {
                 this._coordsFromCurrentGrid = cords => cords;
             }
-            if (!this.utils.Exist() && isBase && maxZoom is not null && extent is not null)
+            if (isBase && extent is null)
             {
-                this.utils.Create(extent.Value, maxZoom.Value, isOneXOne);
+                //throw error if extent is missing in base
+                throw new Exception($"path '{path}' to data does not exist.");
             }
-        }
-
-        public override void Reset()
-        {
-            this.offset = 0;
-        }
-
-        public override void UpdateMetadata(IData data) //TODO: check if should be moved to wrapup?
-        {
-            //TODO: support update old gpkg with known extent 
-            if (data.Type != DataType.GPKG)
+            if (isBase)
             {
-                return;
-            }
-
-            Gpkg gpkg = (Gpkg)data;
-            this.UpdateExtent(gpkg);
-            this.UpdateTileMatrix(gpkg); //TODO: replace with sql based update by zoom level
-        }
-
-        private void UpdateExtent(Gpkg gpkg)
-        {
-            Extent baseExtent = this.utils.GetExtent();
-            Extent extent = gpkg.utils.GetExtent();
-            Extent combinedExtent = new Extent();
-
-            combinedExtent.minX = Math.Min(baseExtent.minX, extent.minX);
-            combinedExtent.minY = Math.Min(baseExtent.minY, extent.minY);
-            combinedExtent.maxX = Math.Max(baseExtent.maxX, extent.maxX);
-            combinedExtent.maxY = Math.Max(baseExtent.maxY, extent.maxY);
+                if (extent is null)
+                {
+                    //throw error if extent is missing in base
+                    throw new Exception($"path '{path}' to data does not exist.");
+                }
 
-            this.utils.UpdateExtent(combinedExtent);
+                if (!this.utils.Exist())
+                {
+                    this.utils.Create(extent.Value, isOneXOne);
+                }
+                else
+                {
+                    this.utils.DeleteTileTableTriggers();
```

Comment:
Why in else? Create also adds all triggers.

Reply:
create dont add those triggers

Reply:
I looked in the code, it seemed like it does. Check again to be sure.

Reply:
it doesn't there are 2 tables that has triggers. it isn't the same triggers